### DISCUSSION TITLE
Kickstarter Demo Feedback

### DISCUSSION THREAD

**Discussion by zicklag** [fishfolk/jumpy#709] [Announcements]
Hello everyone! We've just published our first MVP release! This contains a ton of new stuff including a fully functioning map editor, and the ability to save your own maps.

We need to do a bunch of testing, so testers and bug reports are appreciated! We're ramping up for a Kickstarter very soon so stay tuned!

<hr /><em>This discussion was created from the release <a href='https://github.com/fishfolk/jumpy/releases/tag/v0.6.0'>Kickstarter Demo</a>.</em>

**Comment by etam** [comment]
First impression: The controls settings must be accessible from the pause menu.

My short story of a newbie:
- Launch the game
- Jump straight into new game
- Myself vs bot
- Get wrecked
- How do I play this game?  I can't even pick up the sword.
- Go to pause menu. There surely must be settings or at least something that shows controls.
- Nope. I'm lost and confused. I quit.

Only on a second attempt, while writing this comment, I found that there are controls settings, but only accessible from main menu.

Apart from that I'm quite excited! I had an idea some time ago about a multiplayer game with mods in WASM, but never got time to develop it. So I wish you good luck!

**Reply by erlend-sh** [reply]
Thanks! This lines up with other stories we’ve heard, so I think you’re absolutely right 👌

**Reply by zicklag** [reply]
Seems like the AI might be one of the first things to fix, to improve the game's first impression. Might just need to add random delays or something like that, for now, so that they aren't inescapable.

Pretty much everybody is going to test the game alone first, so the AI is going to have to be better. It might be good to start looking into #675.
